examples/AspNetCoreExample/OriginalDockerfile

```diff
@@ -0,0 +1,13 @@
+# FROM mcr.microsoft.com/dotnet/core/sdk:3.1.201 AS build
```

Comment:
We should delete this if it's no longer in use. I haven't yet checked out your branch but I still do want to support building the docker image via `docker-compose up`.